packages/quantic/scripts/npm/check-sfdx-project.js

```diff
@@ -0,0 +1,37 @@
+const pathlib = require('path');
+const fs = require('fs');
+
+function quanticIsDependency() {
+  const upperDirectories = __dirname.split(pathlib.sep);
+  const firstUpperDirectories = upperDirectories.slice(
+    upperDirectories.length - 5
```

Comment:
for future readers it would be nice if you could put `5` in a constant with a descriptive name.